resources/backupTemplates/nodejs-4.x/templates/templates.json

```diff
@@ -1452,6 +1452,128 @@
             "userPrompt": []
         }
     },
+    {
+        "id": "EventGridBlobTrigger-JavaScript",
```

Comment:
You can remove the non "-4.x" templates, they're not used. This file was originally auto-generated from a custom branch of the templates repo, which is the only reason you see other non "-4.x" templates. Since I assume you didn't auto-generate it again, there's no point in adding them manually

If you're curious, [this code here](https://github.com/microsoft/vscode-azurefunctions/blob/main/src/templates/script/NodeV4Provider.ts#L62) is what filters them out (only the v4 templates will have the triggerType property)

Reply:
Got it. Removed non-4.x templates.